source/Features.rst

```diff
@@ -171,11 +165,11 @@ There is a contextual menu which is at the right side of the card. Once clicked
 Audit Trail
 ^^^^^^^^^^^
 
-We have brought in an  Audit Trail screen for the Administrator to view security-related audits. Click on the “ ≣ “ option which is available on the left side header. You can see the  following details:
+As an Administrator you can use the **Audit Trail** screen to view security-related audits. Click on the “☰” option which is available on the left side header.
    
 .. image:: images/Audit1.png
 
-Click on the **Audit Trail** menu item to navigate to the Audit Trail page.
+Click on the **Audit Trail** menu item. Through this, you can navigate to the Budget details page.
```

Comment:
Audit Trail menu item should say it navigates to Audit Trail page, not Budget Details page.